contrib/runners/mistral/runner.yaml

```diff
@@ -2,8 +2,8 @@
   description: A runner for executing mistral v2 workflow.
   enabled: true
   name: mistral-v2
-  query_module: st2actions.query.mistral.v2
-  runner_module: st2actions.runners.mistral.v2
+  query_module: mistralv2
+  runner_module: mistralv2
```

Comment:
Does the new infrastructure support sub-directories?

E.g. mistral.v2 would map to `<runners path>/mistral/v2.py` or similar.

If not, it would be nice if we can also make support that, but in the worst case we can call it mistral_v2 for now :)